api/db/changelog/db.changelog-115-addColumn-user-create-modified.xml

```diff
@@ -6,15 +6,12 @@
                     http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-1.9.xsd">
   <changeSet author="nsaxena" id="db.changelog-115-addColumn-user-create-modified">
     <addColumn tableName="user">
-      <column name="creation_time" type="datetime">
-        <constraints nullable="true" />
+      <column name="creation_time" type="datetime" defaultValueDate="1970-01-01 00:00:00">
+        <constraints nullable="false" />
       </column>
-      <column name="last_modified_time" type="datetime" defaultValueDate="CURRENT_TIMESTAMP">
-        <constraints nullable="false"/>
+      <column name="last_modified_time" type="datetime" defaultValueDate="1970-01-01 00:00:00">
```

Comment:
I believe the tag we want to use is `valueDate` https://www.liquibase.org/documentation/column.html

That will set the column value when the migration is run. I believe `defaultValueDate` will set the default value in MySQL which we shouldn't need since we should be setting it to the current timestamp on object creation (in our Java code).

Reply:
i was finally able to resolve the script thing and test this, defaultValue results in 1970-01-01 00:00:00 to existing entries

Reply:
![Screen Shot 2020-01-17 at 11 41 34 PM](https://user-images.githubusercontent.com/34481816/72658723-37546c00-3983-11ea-974f-caa835a48188.png)